c_src/scenic/script.c

```diff
@@ -157,30 +157,35 @@ void reset_scripts() {
 //=============================================================================
 // rendering
 
-static inline byte get_byte( void* p, uint32_t offset ) {
+static inline byte get_byte(void* p, uint32_t offset)
+{
   return *((byte*)(p + offset));
 }
 
-static inline unsigned short get_uint16( void* p, uint32_t offset ) {
+static inline unsigned short get_uint16(void* p, uint32_t offset)
+{
   return ntoh_ui16(*((unsigned short*)(p + offset)));
 }
 
-static inline unsigned int get_uint32( void* p, uint32_t offset ) {
+static inline unsigned int get_uint32(void* p, uint32_t offset)
```

Comment:
In the return signature, do we want these to be `uint32_t` instead of `unsigned int`,etc.?